### PR TITLE
 Consolidate test fixtures

### PR DESCRIPTION
**Pull request by aqk**
…ferent implementation

### CODE REVIEW

**Comment by aqk** [comment]
Please see 

https://github.com/Chia-Network/chia-blockchain/pull/10768
https://github.com/Chia-Network/chia-blockchain/pull/10769
https://github.com/Chia-Network/chia-blockchain/pull/10770
https://github.com/Chia-Network/chia-blockchain/pull/10772

File: tests/core/full_node/stores/test_full_node_store.py [hunk 1]
```diff
@@ -47,7 +44,7 @@ def cleanup_keyring(keyring: TempKeyring):
 log = logging.getLogger(__name__)
 
 
-@pytest_asyncio.fixture(scope="function", params=[1, 2])
+@pytest_asyncio.fixture(scope="function", params=[1, 2])  # xxxtest_constants_original.replace
```
Comment (xdustinface) [inline tests/core/full_node/stores/test_full_node_store.py:47]:
```suggestion
@pytest_asyncio.fixture(scope="function", params=[1, 2])
```

Is this intentional?

File: tests/core/full_node/stores/test_full_node_store.py [hunk 2]
```diff
@@ -56,8 +53,8 @@ async def empty_blockchain(request):
     db_path.unlink()
 
 
-@pytest_asyncio.fixture(scope="function", params=[1, 2])
-async def empty_blockchain_original(request):
+@pytest_asyncio.fixture(scope="function", params=[1, 2])  # xxxtest_constants_original.replace
```
Comment (xdustinface) [inline tests/core/full_node/stores/test_full_node_store.py:56]:
```suggestion
@pytest_asyncio.fixture(scope="function", params=[1, 2])
```

Same question :)

**Review by xdustinface — CHANGES_REQUESTED** [review]
See the two comments + i wonder why the GUI update?

File: tests/core/full_node/stores/test_full_node_store.py (hunk 1 above)
Comment (aqk) [reply tests/core/full_node/stores/test_full_node_store.py:46]: No, that's leftover

**Comment by aqk** [comment]
Thank you - the chia-blockchain-gui submodule change was because a change was introduced in github or in the node build process that caused the M1 build test to fail

**Comment by arvidn** [comment]
is this PR still relevant?

**Comment by aqk** [comment]
@arvidn Unfortunately, there's no reply feature like in the code conversation

The reasons it's good to have all the fixtures in `tests/conftest.py`:

* Make it easier for someone to de-duplicate
* Prevent adding duplicate fixtures pre-emptively
* efficiency later: change the scope in one place

**Review by altendky — DISMISSED** [review]
Looks like some extra import changes (isort?) but it's all test files.  The changes appear to all be copy/paste and it looks like tests should break if it were wrong.  Looks like a good step towards deduping all these similar fixtures.

**Comment by aqk** [comment]
@xdustinface those requested changes are addressed.

**Review by altendky — APPROVED** [review]

**Review by xdustinface — APPROVED** [review]

**Pull request merged**